ibis/backends/bigquery/compiler.py

```diff
@@ -698,3 +696,6 @@ def visit_CountDistinct(self, op, *, arg, where):
         if where is not None:
             arg = self.if_(where, arg, NULL)
         return self.f.count(sge.Distinct(expressions=[arg]))
+
+    def visit_RandomUUID(self, op, **kwargs):
```

Comment:
Why do you need to split these out and handle `kwargs`? The existing `SIMPLE_OPS` loop should handle the case of unused arguments.

Reply:
It actually renders the arguments.

Reply:
Ah, right. Extra kwargs get rendered in the `SIMPLE_OPS` loop.

Can you change the `**kwargs` to be `**_` to indicate that they are not used?